async-request-reply/src/AsyncProcessingBackgroundWorker.cs

```diff
@@ -10,19 +10,19 @@ public static class AsyncProcessingBackgroundWorker
     {
         [FunctionName("AsyncProcessingBackgroundWorker")]
         public static void Run(
-            [ServiceBusTrigger("outqueue", Connection = "ServiceBusConnectionAppSetting")]Message myQueueItem, 
+            [ServiceBusTrigger("outqueue", Connection = "ServiceBusConnectionAppSetting")]ServiceBusMessage myQueueItem,
             [Blob("data", FileAccess.ReadWrite, Connection = "StorageConnectionAppSetting")] CloudBlobContainer inputBlob,
```

Comment:
We should probably update this to use the new storage extension - https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs